src/azure-cli/azure/cli/command_modules/util/_help.py

```diff
@@ -31,6 +31,9 @@
   - name: Create a public IP address from body.json file
     text: >
         az rest --method put --uri https://management.azure.com/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/publicIPAddresses/{publicIpAddressName}?api-version=2019-09-01 --body @body.json
+  - name: List the top three resources
+    text: >
+        az rest --method get --uri https://management.azure.com/subscriptions/{subscriptionId}/resources?api-version=2019-07-01 --uri-parameters $top=3
```

Comment:
- `$` will cause problem in Linux bash as it is interpreted as a variable. 
- We can provide another example to use inline parametera in the URL.

Reply:
Yes, `$` works fine in `CMD`, and there will be problems in `Bash` and `PowerShell` that are considered special characters. 
But the escape characters of these environments are different(`bash` is \\ ,`PowerShell` is \` ,`CMD` is ^ ), so what environment is our example mainly based on? Is it `Bash`?

Reply:
We have no standard regarding this. Let's say "In Bash, escape `$` as `\$` and in PowerShell, escape `$` as `` `$ ``".

Reply:
OK~ So I should specify how to escape `$` in different environments in `- name`, right?